### DISCUSSION TITLE
👋 ¡Hola, soy Miguel Ángel Durán! Desde España 🇪🇸

### DISCUSSION THREAD

**Discussion by midudev** [gittogethers/octogatosconf22#48] [General]
¡Hola a todos y todas! 👋

¡Espero que estén muy bien!

Estaré hablando sobre **despliegues a producción en viernes** en la OctogatosConf.

¡Cualquier idea, consejo o pregunta será un placer hablar contigo! 🤗

**Comment by AndreaGriffiths11** [comment]
Gracias! Miguel tambien puedes compartir acerca de tu conferencia por aqui si quieres
